AlexanderOtavka/ride-board
# Create "Me" page

Resolves #40 

# Changes

- Created dedicated "me" page
- Removed "my rides" page
- Restructured layouts and navigation links to make this page first-class in the app
- Removed extra navigation stuff from the welcome page, leaving just the sign in/sign up links

# Manual Testing

1. Check out the welcome page on multiple screen sizes
2. Make sure the sign in/up/out links work
3. Go to the me page in the each app and check it out on multiple screen sizes
5. Make sure the account settings and sign out links work
4. Go to the rides page of both apps and make sure it looks okay
5. Make sure the navigation links look good everywhere
